src/elements/button/button.svelte

```diff
@@ -65,17 +66,22 @@ const handleParentClick = (event: PointerEvent) => {
     aria-label={variant === 'icon' ? label : undefined}
     aria-disabled={isDisabled ? true : undefined}
     {title}
-    class={cx('active:scale-95 motion-safe:transition-transform', {
-      'inline-flex items-center justify-center gap-1.5 py-1.5 px-3 text-xs border':
+    class={cx({
+      'h-[30px] w-[30px]': variant === 'icon',
+      'px-4': !icon || variant === 'icon',
+      'pl-3 pr-4': icon && variant !== 'icon',
+      'inline-flex items-center justify-center gap-1.5 py-1 text-sm border':
         variant !== 'icon',
-      'bg-light border-light hover:bg-medium hover:border-medium':
+      'bg-light border-light hover:bg-medium hover:border-medium active:bg-gray-2':
         variant === 'primary',
-      'bg-gray-9 border-gray-9 text-white hover:bg-black hover:border-black':
+      'bg-gray-9 border-gray-9 text-white hover:bg-black hover:border-black active:bg-[#000]':
```

Comment:
I would add a black or dark variant here rather than using [#000]

```
    backgroundColor: {
      extralight: '#FBFBFC',
      light: '#F7F7F8',
      medium: '#F1F1F4',
    },
```

Reply:
We don't want to use pure black outside of this file